README.md

```diff
@@ -161,6 +161,7 @@ tigmint-make tigmint-long draft=myassembly reads=myreads span=auto G=gsize dist=
 + `mapq=0`: Mapping quality threshold
 + `trim=0`: Number of bases to trim off contigs following cuts
 + `t=8`: Number of threads
++ `ac=3000`: Minimum contig length (bp) for tallying attempted corrections. This is for assessing the accuracy of Tigmint, and will not affect its performance.
```

Comment:
```suggestion
+ `ac=3000`: Minimum contig length (bp) for tallying attempted corrections. This is for logging purposes only, and will not affect the performance.
```
Just to avoid mentioning 'accuracy' in the README